### DISCUSSION TITLE
Mod Ideas

### DISCUSSION THREAD

**Discussion by SomeoneNKnuckles** [SomeoneNKnuckles/Someone-s-Mods#4] [General]
Idk what To Do

**Comment by Mazher94** [comment]
I know you already finished the someone and knuckles chars mod.
But could you fix this bug with Amy galores super sprites being overwritten by sonics sprites?
![Screenshot_20241120_072413](https://github.com/user-attachments/assets/248e1c4b-05ee-4cbf-ad87-7930024395e0)

**Reply by SomeoneNKnuckles** [reply]
it happens without the mod, it's a problem of the MISC & Separete S & H Sprites

**Reply by SomeoneNKnuckles** [reply]
> it happens without the mod, it's a problem of the MISC & Separete S & H Sprites

And You Can Fix It If You Place This [Mod](https://gamebanana.com/mods/476143) Above Amy Galore

**Reply by Mazher94** [reply]
That mod doesn't work with the SHC build. It even causes an error.
